app/ui-react/packages/ui/src/Shared/SynAboutModal.tsx

```diff
@@ -0,0 +1,52 @@
+import { AboutModal } from '@patternfly/react-core';
+import * as React from 'react';
+import { AboutModalContent } from './AboutModalContent';
+
+export interface ISynAboutModal {
+  brandImg: any;
+  handleModalToggle: any;
+  isModalOpen: boolean;
+  trademark: string;
+  productName: string;
+  version: string;
+  buildId: string;
+  commitId: string;
+}
+
+export class SynAboutModal extends React.Component<ISynAboutModal> {
+  constructor(props: ISynAboutModal) {
+    super(props);
+  }
+
+  public render() {
+    const {
+      brandImg,
+      buildId,
+      commitId,
+      isModalOpen,
+      handleModalToggle,
+      productName,
+      trademark,
+      version,
+    } = this.props;
+    return (
+      <React.Fragment>
+        <AboutModal
+          isOpen={isModalOpen}
+          onClose={handleModalToggle}
+          trademark={trademark}
+          brandImageSrc={brandImg}
+          brandImageAlt="Brand Image"
+          productName={productName}
+        >
+          <AboutModalContent
```

Comment:
If you don't put this here and instead just have a `children` prop then you can organize the modal like I had mentioned, where you can do the API call from within the body of the modal, so only when it's shown vs when the page is loaded